src/Components/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import '../Styles/styles.css';
+
+export const GtaVersions = ({ imageLink, versionName, index }) => {
+  const GetLink = (index) => {
```

Comment:
You can remove this function and instead write the return value where it's being called.